parsing/parser.mly

```diff
@@ -1119,6 +1129,31 @@ parse_pattern:
     { $1 }
 ;
 
+parse_mty_longident:
```

Comment:
The convention I follow in my own parsers is `mty_longident_eof` for these starting actions, I personally find it clearer than yours. (The existing code follows no convention anyway.)

Reply:
I was trying to respect the convention used in parse_{core_type,expression,pattern}, but I have no strong opinion on the matter.

Reply:
Ah, I hadn't noticed that this convention is (half-)followed in the existing codebase. It's not a very good convention, but also fairly orthogonal to the present PR; it's fine to leave it as-is.